package/MDAnalysis/coordinates/chain.py

```diff
@@ -561,17 +557,12 @@ def _read_frame(self, frame):
         # update Timestep
         self.ts = self.active_reader.ts
         self.ts.frame = frame  # continuous frames, 0-based
+        self._current_frame = frame
```

Comment:
Possibly make it a double-underscore attribute `__current_frame`? How bad would it be if a user changed it?

Reply:
The next frame ts will start from the newly set value, but the current ts will not change. Just double-underscored it to prevent from confusion.